dubbo-registry/dubbo-registry-nacos/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+###set log levels###
+log4j.rootLogger=info, stdout
+###output to the console###
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=[%d{dd/MM/yy HH:mm:ss:SSS z}] %t %5p %c{2}: %m%n
```

Comment:
Could you please tell us what is your purpose to add `log4j.properties` in test? Will the tests fail if remove this one?

Reply:
it can be removed